src/lib/database.js

```diff
@@ -417,8 +417,10 @@ async function getCollections(page = 1, limit = 100) {
     return response.body.hits.hits.map((r) => (r._source))
   } catch (e) {
     logger.error('Failure getting collections, maybe none exist?', e)
+    return new Error('Collections not found. This is likely '
+    + 'because the server has not been initialized with create_indices, '
+    + 'cannot connect to the database, or cannot authenticate to the database.')
```

Comment:
Would it be helpful to instruct the user what to do if they see this?

Is it a matter of waiting, or are there steps they need to do to rectify this?

Reply:
my presumption here was that they would look for "create_indices" in the readme for the instruction they didn't do, or look in the logs for a message about not being able to connect to the database. I don't think there's much more we can give specific direction on here.